environment.yml

```diff
@@ -10,4 +10,3 @@ dependencies:
   - r==3.4.3
   - gxx_linux-64==7.2.0
   - biopet-bamstats==1.0
-
```

Comment:
I think this file should just be deleted.

Reply:
Very good, I will do that.